Makefile

```diff
@@ -110,7 +110,7 @@ go-mod-cache: go.sum
 	@echo "--> Download go modules to local cache"
 	@go mod download
 
-go.sum: tools go.mod
+go.sum: go.mod
 	@echo "--> Ensure dependencies have not been modified"
 	@go mod verify
```

Comment:
Do we not need `-mod=readonly` here?

Reply:
`-mod=readonly` is not supported by `go mod` just yet - from what I can see here https://github.com/golang/go/issues/31372 and here https://go-review.googlesource.com/c/go/+/174258/, it seems that upstream will include it in Go 1.14